ytfzf

```diff
@@ -639,6 +673,9 @@ shift $((OPTIND-1))
 # Main {{{
 json_file=$(mktemp)
 id_file=$(mktemp)
+#provide easy access to the id, and json
+ln -sf "$id_file" "$ytfzf_selected_ids"
+ln -sf "$json_file" "$ytfzf_video_json_file"
```

Comment:
why not just use this file instead of using the temp file?
as in use the $ytfz_selected_ids file for selected id and so on.
Since we anyways have to mess with pid for both ways

Reply:
> why not just use this file instead of using the temp file?

fixed